ResearchKit/Common/ORKTask.h

```diff
@@ -107,6 +108,13 @@ ORK_AVAILABLE_DECL
  */
 @property (nonatomic, copy, readonly) NSString *identifier;
 
+/**
+ The theme for the task.
+ 
+ Various UI elements may check the theme and use it to apply modifications.
+ */
+@property (nonatomic, retain, nullable) CEVRKTheme *cev_theme;
```

Comment:
Actually, I'm seeing a compilation problem with a [custom ORKTask class](https://github.com/CareEvolution/RKStudio-Participant/blob/7cb7be8a095c2b8db5779348a0b88f490ab2a66b/RKStudio-Participant/ResearchKit/TaskManager.swift#L120), implemented in Swift, in RKStudio-Participant. At this line, `cev_theme` is declared as a nullable property, but in CEVRKThemedUIElement, `cev_theme` is declared as a non-null func. Since ORKTask is declared as conforming to CEVRKThemedUIElement, Swift can't reconcile these differences when attempting to declare a class conforming to ORKTask, and you get an unresolvable compilation error.

Reply:
I changed to protocol to nullable since the theme checking logic can handle null.

Reply:
The Swift `class Task: ORKTask { ... }` still doesn't compile, because in Swift, the CEVRKThemedUIElement protocol wants `cev_theme` to be a func, but in the ORKTask protocol, it wants `cev_theme` to be a var.

Reply:
I apologize for not testing in RKStudio-Participant. Changing explicitly to a property seems to fix this.

Reply:
No problem, I initially hadn't tested it either. Shows how much more lenient Objective-C is with these declarations. Everything looks good/compiles now 🚢